build/ci/release.yml

```diff
@@ -1,7 +1,7 @@
 variables:
   - &go_linux_version
-      go_root: "/opt/golang/go1.21.3"
-      go_bin: "/opt/golang/go1.21.3/bin"
+      go_root: "/opt/golang/go1.21"
+      go_bin: "/opt/golang/go1.21/bin"
       go_base_path: ""
   - &go_windows_version
       go_root: "c:\\golang\\go1.19"
```

Comment:
seems like windows is still using go 1.19

Reply:
I forgot to ask you. In the last bump we did not moved it to 1.20?:
https://github.com/mongodb/mongodb-atlas-cli/pull/2085/files

Was there any reason for it?

Reply:
probably overlooked